src/sentry/lang/javascript/processor.py

```diff
@@ -578,15 +614,16 @@ def expand_frames(self, frames, release):
                     })
 
                 # Store original data in annotation
+                # HACK(dcramer): we stuff things into raw which gets popped off
+                # later when adding the raw_stacktrace attribute.
+                raw_frame = frame.to_json()
                 frame.data = {
-                    'orig_lineno': frame.lineno,
```

Comment:
lol

![image](https://cloud.githubusercontent.com/assets/2153/15621051/07be84fa-2413-11e6-9f3d-ee74094e2b40.png)

It seems we should remove it now though, right? Seeing as you can toggle between original line number, file, etc.